src/client/Microsoft.Identity.Client/AppConfig/AuthorityInfo.cs

```diff
@@ -428,6 +441,12 @@ private static AuthorityType GetAuthorityType(string authority)
             return AuthorityType.Aad;
         }
 
+        private static bool isCiamAuthority(string authority)
+        {
+            var uri = new Uri(authority);
+            return uri.Host.EndsWith(".ciamlogin.com");
```

Comment:
Let's use a constant all over the code. The URL could change.
cc: @bgavrilMS

Reply:
Yes, makes sense